client/tests/components/form/MultiStepButtons.test.tsx

```diff
@@ -0,0 +1,197 @@
+    expect(screen.getByRole("button", { name: "Cancel" })).not.toBeDisabled();
+    expect(screen.queryByRole("button", { name: "Next" })).not.toBeDisabled();
+    expect(
+      screen.queryByRole("button", { name: "Save and Continue" }),
+    ).not.toBeInTheDocument();
+  });
+
+  it("shows CAS users correct buttons on middle steps", () => {
+    vi.resetAllMocks();
+    useSession.mockReturnValue({
+      data: {
+        user: {
+          app_role: "cas_admin",
+        },
+      },
+    } as Session);
+    render(<MultiStepButtons {...defaultProps} step={1} />);
+    expect(screen.getByRole("button", { name: "Back" })).not.toBeDisabled();
+    expect(screen.getByRole("button", { name: "Cancel" })).not.toBeDisabled();
+    expect(screen.queryByRole("button", { name: "Next" })).not.toBeDisabled();
+    expect(
+      screen.queryByRole("button", { name: "Save and Continue" }),
+    ).not.toBeInTheDocument();
+  });
+
+  it("shows CAS users correct buttons on final step", () => {
+    vi.resetAllMocks();
+    useSession.mockReturnValue({
+      data: {
+        user: {
+          app_role: "cas_admin",
+        },
+      },
+    } as Session);
+    render(<MultiStepButtons {...defaultProps} step={2} />);
+    expect(screen.getByRole("button", { name: "Back" })).not.toBeDisabled();
+    expect(screen.getByRole("button", { name: "Cancel" })).not.toBeDisabled();
+    expect(
+      screen.queryByRole("button", { name: "Save and Continue" }),
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole("button", { name: "Submit" }),
+    ).not.toBeInTheDocument();
+  });
+
+  it("does not show the Back button when allowBackNavigation is false", () => {
+    render(<MultiStepButtons {...defaultProps} allowBackNavigation={false} />);
+
+    expect(
+      screen.queryByRole("button", { name: "Back" }),
+    ).not.toBeInTheDocument();
+  });
+
+  it("Back button has correct link", () => {
+    render(<MultiStepButtons {...defaultProps} step={2} />);
+    expect(screen.getByRole("link", { name: "Back" })).toHaveAttribute(
+      "href",
+      "anything.com/2",
+    );
+  });
+  it("Cancel button link is correct", () => {
+    render(<MultiStepButtons {...defaultProps} />);
+
+    expect(screen.getByRole("link", { name: "Cancel" })).toHaveAttribute(
+      "href",
+      "https://www.test.com/",
+    );
+  });
+  it("Next button has correct link", async () => {
+    render(<MultiStepButtons {...defaultProps} />);
+
+    expect(screen.getByRole("link", { name: "Next" })).toHaveAttribute(
+      "href",
+      "anything.com/2",
+    );
+  });
+  it("Save and Continue has correct link", () => {
+    render(<MultiStepButtons {...defaultProps} step={1} />);
+
+    expect(screen.getByRole("link", { name: "Next" })).toHaveAttribute(
+      "href",
+      "anything.com/3",
+    );
+  });
+  it("Submit button navigates correctly", () => {
+    // brianna can we actually test this without the form?
```

Comment:
thoughts?

Reply:
It's tested in the form component test with a mock test schema, yes. I don't think we need to test it in this component because it's not doing anything here (unlike something like the cancel button, which I can test has the correct link) but wanted a second opinion

Reply:
Oh sorry that makes sense, I forgot we handle that stuff in say `OperationsForm`. Seems out of scope for this test 👍